packages/mobx/__tests__/v5/base/errorhandling.js

```diff
@@ -4,7 +4,7 @@ const utils = require("../../v5/utils/test-utils")
 
 const { observable, computed, $mobx, autorun } = mobx
 
-const voidObserver = function () { }
```

Comment:
I didn't change this on purpose, I think the formatter is just getting excited because I changed other things in this file